mParticle-OneTrust/MPKitOneTrust.m

```diff
@@ -1,4 +1,5 @@
 #import "MPKitOneTrust.h"
+#import <OTPublishersHeadlessSDK/OTPublishersHeadlessSDK-Swift.h>
```

Comment:
This might be another one that isn't needed if we're not referencing the OTSDK in the plugin itself.